airbyte-integrations/connectors/destination-chroma/destination_chroma/utils.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import re
+from ipaddress import ip_address
+
+
+def validate_collection_name(stream_name: str):
+    if is_valid_collection_name(stream_name):
```

Comment:
The check is good, but please let's not try to fix it automatically - if the provided collection name is not valid, don't fix it on the fly, but fail the `check` of the connector so the user can fix it.